pkgs/development/compilers/rust/rustc.nix

```diff
@@ -59,6 +59,7 @@ in stdenv.mkDerivation rec {
     cxxForHost = "${pkgsBuildHost.targetPackages.stdenv.cc}/bin/${pkgsBuildHost.targetPackages.stdenv.cc.targetPrefix}c++";
     ccForTarget  = "${pkgsBuildTarget.targetPackages.stdenv.cc}/bin/${pkgsBuildTarget.targetPackages.stdenv.cc.targetPrefix}cc";
     cxxForTarget = "${pkgsBuildTarget.targetPackages.stdenv.cc}/bin/${pkgsBuildTarget.targetPackages.stdenv.cc.targetPrefix}c++";
+    toFlag = x: if x then "true" else "false";
```

Comment:
We don't need to define this function here — there's already `lib.boolToString`.

Reply:
Thanks, fixed